keda/templates/26-metrics-podmonitor.yaml

```diff
@@ -9,21 +9,21 @@ metadata:
     app.kubernetes.io/part-of: {{ .Values.operator.name }}
     app.kubernetes.io/managed-by: {{ .Release.Service }}
     app.kubernetes.io/instance: {{ .Release.Name }}
-    {{- range $key, $value := .Values.prometheus.podMonitor.additionalLabels }}
+    {{- range $key, $value := .Values.prometheus.metricServer.podMonitor.additionalLabels }}
     {{ $key }}: {{ $value | quote }}
     {{- end }}
-  {{- with .Values.prometheus.podMonitor.namespace }}
+  {{- with .Values.prometheus.metricServer.podMonitor.namespace }}
   namespace: {{ . }}
   {{- end }}
 spec:
   podMetricsEndpoints:
```

Comment:
Shouldn't we add the operator as an endpoint here?

Reply:
I think we would like to separate between the `operator` and the `metricServer`

Reply:
Do you mean in terms of the pod monitor or? Because that's ok for me but we are not adding a new one for the operator itself.

Reply:
For pod monitor yes.

Reply:
Can you add the other pod monitor then, please?

Reply:
I didn't changed the existing one, just the `values.yaml` and accordingly needed to change the names under the `podMonitor` (it will break anyways)

Reply:
If we don't add pod monitor support for the operator, I'm not sure why we have to break the current configuration then?

I think I'm not following what you are saying, sorry about that :/

Reply:
Sorry. I now understand that I didn't upload the `podmonitor` for the operator. It has been added. Sorry for the confusion.

Reply:
No problem at all, thanks for adding!